akka-stream/src/main/scala/akka/stream/ActorFlowMaterializer.scala

```diff
@@ -139,6 +138,11 @@ abstract class ActorFlowMaterializer extends FlowMaterializer {
    */
   private[akka] def actorOf(props: Props, name: String): ActorRef
 
+  /**
+   * INTERNAL API
+   */
+  private[akka] def system: ActorSystem
```

Comment:
would it also be interesting to have this materializer’s “guardian” actorRef available so that the logged names are unique?